src/components/TaskCard/EditTaskCard.jsx

```diff
@@ -0,0 +1,89 @@
+import React, { useState } from "react";
+import { useDispatch } from "react-redux";
+import PropTypes from "prop-types";
+import { sanitizeText } from "utils/helpers/sanitizeText.js";
+import { editTask, completeTask } from "store/actions/";
+import style from "components/TaskCard/index.module.scss";
+import { DELETE, DELETE_ALT, ENTER } from "utils/constant";
+import { COMPLETE, COMPLETE_ALT } from "utils/constant/images";
+const EditTaskCard = ({ id, task, setEditableTask }) => {
+  const [inputText, setInputText] = useState(task);
+  const [error, setError] = useState("");
+  const dispatch = useDispatch();
+  const handleInputText = (e) => {
+    setInputText(e.target.value);
+  };
+
+  const storeTask = () => {
+    const sanatizedTask = sanitizeText(inputText);
+    if (sanatizedTask === "") {
+      setError("Please add task description");
+      return;
+    }
+    setEditableTask(null);
+    dispatch(
+      editTask({
+        id,
+        task: sanatizedTask,
+      })
+    );
+    setInputText(null);
+  };
+
+  const cancelAction = () => {
+    setEditableTask(null);
+  };
+
+  const completeAction = () => {
+    storeTask();
+
+    dispatch(completeTask(id));
+  };
+
+  const storeTaskOnEnter = (e) => {
+    if (e.key === ENTER) {
+      e.preventDefault();
+      storeTask();
+    }
+  };
+
+  const sendCursorToEnd = (e) => {
+    const inputElement = e.target;
+    inputElement.selectionStart = inputElement.value.length;
+    inputElement.selectionEnd = inputElement.value.length;
+  };
```

Comment:
```suggestion
const sendCursorToEnd = (e) => {
  const inputElement = e.target;
  inputElement.setSelectionRange(inputElement.value.length, inputElement.value.length);
};
```
Can you please give it a try?